powerauth-data-adapter-model/src/main/java/io/getlime/security/powerauth/lib/dataadapter/model/request/DecorateOperationFormDataRequest.java

```diff
@@ -14,34 +14,34 @@
  * limitations under the License.
  */
 
-package io.getlime.security.powerauth.lib.dataadapter.model.response;
+package io.getlime.security.powerauth.lib.dataadapter.model.request;
 
-import io.getlime.security.powerauth.lib.dataadapter.model.entity.BankAccountList;
+import io.getlime.security.powerauth.lib.dataadapter.model.entity.OperationContext;
 
 /**
- * Response with list of user bank accounts.
+ * Request object for obtaining bank accounts by user ID.
```

Comment:
I think this comment is no longer accurate, since this is now a generic form data decoration request.